### PR TITLE
[ES] Add `G` missing stubs (3)

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

Missing G stubs from the `Missing stubs (up-to-date in English)` section according to the osu! wiki status in spanish.

28 stubs in total, divided into 4 pull requests of 7 stubs each. Continued in: #10627

~~Let me know if there are any problems with the image because of its _Christmas_ design~~

### CODE REVIEW

File: wiki/Gameplay/Input_device/Touch_device/es.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+---
+
+# Dispositivo táctil
+
+*Véase también: [Estilos de juego](/wiki/Gameplay/Play_style)*
+
+Un **dispositivo táctil** es un dispositivo con pantalla táctil que permite al usuario interactuar con la computadora usando sus dedos. osu! es compatible con dispositivos táctiles de forma nativa. Puedes tocar la pantalla para apuntar y hacer clic al mismo tiempo. Los dispositivos táctiles suelen venir en forma de portátiles con pantalla táctil, tabletas gráficas de gama alta o teléfonos inteligentes.
```
Comment (agatemosu) [inline wiki/Gameplay/Input_device/Touch_device/es.md:9]:
```suggestion
Un **dispositivo táctil** es un dispositivo con pantalla táctil que permite al usuario interactuar con el ordenador usando sus dedos. osu! es compatible con dispositivos táctiles de forma nativa. Puedes tocar la pantalla para apuntar y hacer clic al mismo tiempo. Los dispositivos táctiles suelen venir en forma de portátiles con pantalla táctil, tabletas gráficas de gama alta o teléfonos inteligentes.
```

File: wiki/Gameplay/Input_device/es.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+stub: true
+---
+
+# Dispositivo de entrada
+
+Un **dispositivo de entrada** es un dispositivo que se usa para controlar la entrada en una computadora o un software específico. En la osu! wiki, este término se refiere específicamente a dispositivos que pueden usarse en osu!.
```
Comment (agatemosu) [inline wiki/Gameplay/Input_device/es.md:7]:
```suggestion
Un **dispositivo de entrada** es un dispositivo que se usa para controlar la entrada en un ordenador o un software específico. En la osu! wiki, este término se refiere específicamente a dispositivos que pueden usarse en osu!.
```

File: wiki/Gameplay/Judgement/Combobreak/es.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+tags:
+  - miss
+  - combo miss
+  - fallo
+  - fallo de combo
+---
+
+# Combobreak
+
+**Combobreak** es un término usado para describir a un jugador que pierde su [combo](/wiki/Gameplay/Combo_(score_multiplier)) durante cualquier parte de un [beatmap](/wiki/Beatmap). De forma predeterminada, se reproducirá un «sonido de ruptura de combo» si el jugador ha roto un combo de 21x o superior. Y si está habilitado en la configuración, aparecerá una imagen de combo burst en ciertos [hitos](/wiki/Gameplay/Combo_milestone).
```
Comment (agatemosu) [inline wiki/Gameplay/Judgement/Combobreak/es.md:12]:
```suggestion
El **combobreak** es un término usado para describir a un jugador que pierde su [combo](/wiki/Gameplay/Combo_(score_multiplier)) durante cualquier parte de un [beatmap](/wiki/Beatmap). De forma predeterminada, se reproducirá un «sonido de ruptura de combo» si el jugador ha roto un combo de 21x o superior. Y si está habilitado en la configuración, aparecerá una imagen de combo burst en ciertos [hitos](/wiki/Gameplay/Combo_milestone).
```

File: wiki/Gameplay/Mind_block/es.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  - term
+  - bloqueo mental
+  - término
+---
+
+# Mind block
+
+*No debe confundirse con [Notelock](/wiki/Gameplay/Judgement/Notelock) o [Fingerlock](/wiki/Gameplay/Fingerlock).*
+
+**Mind block** es un término usado para describir cuando un jugador pierde o falla repetidamente en lo que antes era una sección fácil de un [beatmap](/wiki/Beatmap), a menudo como resultado de volver a intentar dicho beatmap repetidamente. El término se llama «mind block» ya que el evento que describe crea la sensación de que el cerebro (mente) del jugador ha sido, en sentido figurado, «bloqueado» para no obtener con éxito un full combo o completar la sección antes mencionada.
```
Comment (agatemosu) [inline wiki/Gameplay/Mind_block/es.md:16]:
```suggestion
El **mind block** es un término usado para describir cuando un jugador pierde o falla repetidamente en lo que antes era una sección fácil de un [beatmap](/wiki/Beatmap), a menudo como resultado de volver a intentar dicho beatmap repetidamente. El término se llama «mind block» ya que el evento que describe crea la sensación de que el cerebro (mente) del jugador ha sido, en sentido figurado, «bloqueado» para no obtener con éxito un combo completo o completar la sección antes mencionada.
```

File: wiki/Gameplay/Mind_block/es.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+## Causas
+
+El mind block a menudo es causado por una gran acumulación de «[memoria muscular](https://en.wikipedia.org/wiki/Muscle_memory) inútil» debido al reintento constante, repetido y rápido de un beatmap. Aunque no se ha realizado ninguna ciencia o investigación oficial sobre este fenómeno específico en osu!, la comunidad cree ampliamente que durante el mind block, la mente del jugador a menudo se aburre, se agota o se cansa de hacer los mismos patrones y escuchar los mismos pocos segundos de música, que le cuesta concentrarse y prestar atención y, por lo tanto, le cuesta desarrollar una nueva memoria muscular.
+
+## Remedios y prevención
+
+Muchos jugadores suelen remediar y prevenir el mind block tomando descansos o jugando diferentes beatmaps, ya que es un consenso común entre los jugadores de osu! que si le dan un descanso a la mente de osu! le permitirá descansar, reflexionar y recuperarse. Del mismo modo, muchos jugadores suelen cambiar de beatmap después de tantos intentos *antes* de que se produzca el mind block para permitir centrarse mejor en otros beatmaps.
```
Comment (agatemosu) [inline wiki/Gameplay/Mind_block/es.md:24]:
```suggestion
Muchos jugadores suelen remediar y prevenir el mind block tomando descansos o jugando diferentes beatmaps, ya que es un consenso común entre los jugadores de osu! que dar una pausa a la mente de osu! permitirá descansar, reflexionar y recuperarse. Del mismo modo, muchos jugadores suelen cambiar de beatmap después de tantos intentos *antes* de que se produzca el mind block para permitir centrarse mejor en otros beatmaps.
```

File: wiki/Gameplay/Pass/es.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+---
+stub: true
+tags:
+  - passing
+  - pasar
+---
+
+# Pass
+
+Un **pass** describe la finalización de un [beatmap](/wiki/Beatmap) o una determinada sección de un beatmap sin el uso de [mods](/wiki/Gameplay/Game_modifier) ​​que previenen perder (por ejemplo, [No Fail](/wiki/Gameplay/Game_modifier/No_Fail)), o llegar a la sección de [descanso](/wiki/Beatmap/Break) con más del 50 % de [salud](/wiki/Gameplay/Health). En el caso de llegar a una sección de descanso, se mostrará un indicador con un sonido para indicar que el jugador ha pasado la sección anterior.
```
Comment (agatemosu) [inline wiki/Gameplay/Pass/es.md:10]:
```suggestion
Un **pass** describe la finalización de un [beatmap](/wiki/Beatmap) o una determinada sección de un beatmap sin el uso de [mods](/wiki/Gameplay/Game_modifier) ​​que previenen perder (p. ej., [No Fail](/wiki/Gameplay/Game_modifier/No_Fail)), o llegar a la sección de [descanso](/wiki/Beatmap/Break) con más del 50 % de [salud](/wiki/Gameplay/Health). En el caso de llegar a una sección de descanso, se mostrará un indicador con un sonido para indicar que el jugador ha pasado la sección anterior.
```

File: wiki/Gameplay/Pass/es.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+---
+stub: true
+tags:
+  - passing
+  - pasar
+---
+
+# Pass
+
+Un **pass** describe la finalización de un [beatmap](/wiki/Beatmap) o una determinada sección de un beatmap sin el uso de [mods](/wiki/Gameplay/Game_modifier) ​​que previenen perder (p. ej., [No Fail](/wiki/Gameplay/Game_modifier/No_Fail)), o llegar a la sección de [descanso](/wiki/Beatmap/Break) con más del 50 % de [salud](/wiki/Gameplay/Health). En el caso de llegar a una sección de descanso, se mostrará un indicador con un sonido para indicar que el jugador ha pasado la sección anterior.
```
Comment (agatemosu) [inline wiki/Gameplay/Pass/es.md:10]: Perdóneme, veo que en ningún artículo pone "," después de "p.ej.".
```suggestion
Un **pass** describe la finalización de un [beatmap](/wiki/Beatmap) o una determinada sección de un beatmap sin el uso de [mods](/wiki/Gameplay/Game_modifier) ​​que previenen perder (p. ej. [No Fail](/wiki/Gameplay/Game_modifier/No_Fail)), o llegar a la sección de [descanso](/wiki/Beatmap/Break) con más del 50 % de [salud](/wiki/Gameplay/Health). En el caso de llegar a una sección de descanso, se mostrará un indicador con un sonido para indicar que el jugador ha pasado la sección anterior.
```

**Review by agatemosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**